src/components/nav/Nav.js

```diff
@@ -1,31 +1,34 @@
 import React, { Component } from 'react';
 import './Nav.css';
 import { NavLink } from 'react-router-dom';
+import { Container, Row, Col } from 'reactstrap';
+
 // import images
-import imgHeader from '../../images/shifraLogo.png';
+import SiteLogo from '../logo/Logo';
+import Searchbar from '../searchbar/SearchBar';
+import LanguageButton from '../languageButton/languagebutton';
 
 class Nav extends Component {
```

Comment:
Im not a fan of short hand naming. The more expressive the better. Can we change this to "Navigation"?